internal/browser/pool.go

```
@@ -0,0 +1,154 @@
+	for name, s := range sessions {
+		if err := s.bctx.Close(); err != nil {
+			log.Printf("[browser] close ctx for %q: %v", name, err)
+		}
+	}
+	if err := p.browser.Close(); err != nil {
+		log.Printf("[browser] close browser: %v", err)
+	}
+	return p.pw.Stop()
+}
+
+func (p *Pool) getOrCreatePage(user string) (playwright.Page, error) {
```

golangci-lint (2.12.2)

[error] func (*Pool).getOrCreatePage is unused

(unused)